package/kernel/linux/modules/block.mk

```diff
@@ -501,7 +501,7 @@ define KernelPackage/nvme
 	CONFIG_NVME_CORE \
```

Comment:
And to do this, replace `DEPENDS:=@PCI_SUPPORT` by `DEPENDS:=@PCI_SUPPORT +kmod-hwmon-core`.

Reply:
Oh, I wonder why it works fine locally, thank you! I'll update it tomorrow :)

Reply:
Probably because target you are testing against already has hwmon enabled